company.c

```diff
@@ -0,0 +1,38 @@
+#include <stdio.h>
+#include <string.h>
+#include <math.h>
+#include <stdlib.h>
+
+struct employee {
+	char name [100];
+	char position [100];
+	long int salary;
```

Comment:
Nice job, but you didn't follow directions completely - you should have had salaries in decimal format!